src/pages/home/home.html

```diff
@@ -2,9 +2,11 @@
   <img id="profile-icon" [src]="gravatarUrl || 'assets/profile.jpg'" on-tap="presentProfilePage()" />
   <div class="button-group">
     <div *ngIf="isSpaceCreated" class="location-button">
+      <!--REMOVE THIS BUTTON AND GROUP TEST SERVICE FOR PRODUCTION-->
+      <button ion-button color="secondary" round (click)="groupTestService.generateRandomGeoUser(latitude, longitude, groupSocketService.userInfo.groupUID)">Generate user</button>
```

Comment:
Comment this out, we don't want this in master. When we are developing, we can uncomment it and test.